com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -758,6 +752,13 @@ public void SetLogLevel(LogCategory Category, LogLevel Level)
             /// </summary>
             private void InitializeLogLevels()
             {
+                // TODO: Restore original implementation for this function, but
```

Comment:
This should be wrapped in an android specific conditional compile.

Reply:
This sounds like there's a known issue with Android using SetLogLevel. Does it not work with it? Can people still define custom log levels for Android?

Reply:
The issue with SetLogLevel is unique to Android, but resolving it requires refactoring some of the LogLevel code - which is outside the scope of this PR.

Functionality has been changed so that it uses an Android specific conditional for compile time, and I'll add a task to address it for Android specifically at a later time.